### DISCUSSION TITLE
ESP32_DevkitC_v4    (wroom-32u)

### DISCUSSION THREAD

**Discussion by mmmmm009** [arendst/Tasmota#20871] [Q&A] [answered]
Hi,

Is ESP32_DevkitC_v4  supported by TASMOTA?

I managed to flash the latest version of tasmota (online flashing tool) and then Update to a different bin file (tasmocompiler) with Script enabled. However ESP32 seems to be unstable.

Which ver of ESP should I choose for ESP32_DevkitC_v4 ?
![tasmo setting](https://github.com/arendst/Tasmota/assets/72743330/08bed9d9-e512-4f73-baca-41667a06ab3c)

**Comment by Jason2866** [comment, marked as answer]
Choose ESP32 generic

**Reply by sfromis** [reply]
The board selection is not about what your dev board has of silk screen, but which processor series is on there. "Typical" ESP32 boards are indeed generic, unless they clearly talk about ESP32-S2, ESP32-S3, ESP32-C3 etc. In general, it will not work to pick a different series.
